tools/link.py

```diff
@@ -2835,16 +2834,22 @@ def process_libraries(state):
     if js_libs is not None:
       continue
 
-    path = None
-    for suff in suffixes:
-      name = 'lib' + lib + suff
-      path = find_library(name, state.lib_dirs)
-      if path:
-        break
+    if not settings.RELOCATABLE:
```

Comment:
Why was there no RELOCATABLE check before? Was it not needed?

Reply:
(maybe related to the test failures?)

Reply:
Indeed, we were needlessly expanding all `-l` flags even though wasm-ld does this.